osu.Game/Localisation/SongSelectStrings.cs

```diff
@@ -19,6 +19,46 @@ public static class SongSelectStrings
         /// </summary>
         public static LocalisableString LocallyModifiedTooltip => new TranslatableString(getKey(@"locally_modified_tooltip"), @"Has been locally modified");
 
+        /// <summary>
+        /// "General"
+        /// </summary>
+        public static LocalisableString General => new TranslatableString(getKey(@"general"), @"General");
```

Comment:
Not sure about having "general" in so many places for translation.

https://github.com/ppy/osu/blob/95a9b532dfc8feff57e524a2361798a6968ff2e9/osu.Game/Localisation/GameplaySettingsStrings.cs#L25

is at least one other example.

Maybe these should be in the general localisations as a single string?